custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -58,8 +58,47 @@
 {% endcomment %}
         <script src="{% static 'js/topojsons/states_v2.topojson.js' %}"></script>
         <script src="{% static 'js/topojsons/districts_v2.topojson.js' %}"></script>
+
+        {# to be removed once  once we move the topojson change from feature flag #}
         <script src="{% static 'js/topojsons/blocks_v3.topojson.js' %}"></script>
 
+        <script src="{% static 'js/topojsons/blocks/maharashtra_blocks_v3.topojson.js' %}"></script>
```

Comment:
Do we have to load all of these up front? It seems like it would be much better if we loaded the individual state's data asynchronously when it is needed and not require downloading what I think is like an additional 10MB or more when you load any dashboard page...

Without knowing why we are doing this or exactly how/when the data is used it's a little unclear to me how hard the above would be or whether it makes sense.